eZ/Publish/SPI/FieldType/FieldStorage.php

```diff
@@ -16,16 +14,7 @@
  *
  * Methods in this interface are called by storage engine.
  *
- * $context array passed to most methods provides some context for the field handler about the
- * currently used storage engine.
- * The array should at least define 2 keys :
- *   - identifier (connection identifier)
- *   - connection (the connection handler)
- * For example, using Legacy storage engine, $context will be:
- *   - identifier = 'LegacyStorage'
- *   - connection = {@link \eZ\Publish\Core\Persistence\Doctrine\ConnectionHandler} object handler (for DB connection),
- *                  to be used accordingly to
- *                  {@link http://incubator.apache.org/zetacomponents/documentation/trunk/Database/tutorial.html ezcDatabase} usage
+ * $context array passed to most methods is deprecated and will be dropped in the next major version.
```

Comment:
Isn't now the best time to drop it, and push deprecation to current LTS?

Reply:
Unfortunately would require SPI breaking changes to ~36 methods in our OSS code, not to mention  1st party packages providing Field Types. Planned to do it as a follow-up, but it's too late now :( Gonna add a BC note though.